src/scaffolding/master.scaffolding.hbs

```diff
@@ -10,9 +9,8 @@
                 One web component library. As many worlds as you want. Decide what the future looks like.
             </p>
             <!-- placeholder buttons - will be replaced with bio components -->
-            <bio-button title="Explore" data-resources="[{paths: ['components/BioButton/index.js']}]"></bio-button>
-            <bio-button title="See Docs" modifier="blue" data-resources="[{paths: ['components/BioButton/index.js']}]">
-            </bio-button>
+            <button class="splash__button splash__button--orange">Explore</button>
```

Comment:
This has already been replaced by `bio-button`s and should create a conflict when you merge develop. These were just placeholders.